dev-docs/source/Testing/MuonAnalysis_test_guides/Muon Analysis Unscripted Testing Guide.rst

```diff
@@ -8,46 +8,85 @@ Muon Interfaces Unscripted Testing
 
 Preamble
 ^^^^^^^^^
-This document is intended for developers to use for unscripted testing of `Muon Analysis` and
-`Frequency Domain Analysis` GUI's.
+This document is intended for developers to use for unscripted testing of muon GUI's.
 
 The tests follow real use cases provided by scientists and are intended to exercise all the interface's functionality.
 As changes are made to the interface and features added, anything for which it is not possible to write an automated
 test should have a manual test added to this list.
 
-.. note:: The tests here are grouped into four sections. The test groups should be run in order and, within each test
-   group, the individual tests should also be run in order. This is because, to avoid duplication, some tests rely on
-   previous tests having been run first.
+.. note:: The tests here are grouped into sections. The test groups can be done in any order.
+
 
 Common setup
 ^^^^^^^^^^^^
 - Set your facility to ISIS
-- Ensure the files ``EMU00020918-20``, ``MUSR00015189`` and ``MUSR00062260-1`` are in Mantid's path
-- Open :menuselection:`Interfaces --> Muon --> Muon Analysis`
-- At this point, before loading any data, the *Grouping Options* and *Data Analysis* tabs should be disabled.
+- Ensure you can acces the files for the test (listed below).
+
+
+Group 1: MUSR data
+^^^^^^^^^^^^^^^^^^
+
+This group tests data from the MUSR instrument and provides an introduction to the data these tests will be working with (groups and pairs).
+The testing instructions can be found at :ref:`Muon_Analysis_MUSR-ref`.
+
+You will need the following runs:
+
+- MUSR 62250-1
+- MUSR 62260
```

Comment:
Could we put file names instead for all of these? That way we can simply search for the filename instead of trying to guess if it's .nxs/.raw/.something

Reply:
The data will only be nxs. I have used this convention as it is what is used in the GUI